tutorials/quickstarts/evals_quickstart.ipynb

```diff
@@ -0,0 +1,500 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    eval_model = OpenAIModel(model="gpt-4-turbo-preview")
</pre><p>Change to GPT4o</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/4371/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>